client/js/lounge.js

```diff
@@ -1039,9 +1042,7 @@ $(function() {
 			socket.emit("names", {target: self.data("id")});
 		}
 
-		if (screen.width > 768 && chan.hasClass("chan")) {
-			input.focus();
-		}
+		focus();
```

Comment:
I was concerned for a bit but just clarifying after testing: if this follows the same pattern than the nick-cycler, programmatically, it's re-opening indeed, but in practice, there is no flickering of the keyboard, it stays open.

I have just been playing with The Lounge and other messaging apps, and I definitely see this is as a UX improvement.